ansible_bender/core.py

```diff
@@ -173,6 +174,29 @@ def _get_path_our_site(self):
         # hence, let's add the site ab is installed in to sys.path
         return os.path.dirname(os.path.dirname(ansible_bender.__file__))
 
+    def _correct_host_entries(self, playbook_path, tmpDir):
+        """ Correct the host entries in the playbook and all imported playbooks """
+        tmp_pb_path = os.path.join(tmpDir, str(uuid.uuid4()) + ".yaml")
+
+        with open(playbook_path, "r") as fd_r:
+            pb_dict = yaml.safe_load(fd_r)
+            
+        for idx, doc in enumerate(pb_dict):
+            imported_playbook = doc.get("import_playbook")
+            if imported_playbook:
+                import_base_path = os.path.dirname(playbook_path)
+                imported_playbook_path = os.path.join(import_base_path, imported_playbook)
+                doc["import_playbook"] = self._correct_host_entries(imported_playbook_path, tmpDir)
```

Comment:
would some logger.debug make sense here as well? especially when it's called recursively

Reply:
Made a simple one in [2abc606](https://github.com/ansible-community/ansible-bender/pull/268/commits/2abc6065f4735e122fda3513b9cfe32b7ade51f3) 🙂  Does it look okay?

Reply:
yup, looks fine!